Userland/Libraries/LibVT/TerminalWidget.cpp

```diff
@@ -87,7 +87,7 @@ TerminalWidget::TerminalWidget(int ptm_fd, bool automatic_size_policy)
     set_pty_master_fd(ptm_fd);
 
     on_emoji_input = [this](auto emoji) {
-        inject_string(emoji);
+        emit(emoji.bytes().data(), emoji.length());
```

Comment:
oooh boy, i think i got lucky here...

i didn't know that `emoji.bytes().size()` is equal to `emoji.length()`. :eyes:

i need to remember that `StringView` is just a pointer and a size with no encoding.

Reply:
I doubt we're gonna ever add non-byte length under such an innocuous name, so this seems fine.